### PR TITLE
Create the finance vertical page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Create the finance vertical page

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/financial-services](http://0.0.0.0:8001/financial-services)
- Check the page looks ok
- Check the copy is the same as the [copy doc](https://docs.google.com/document/d/1o9s_MFaiZZIS26tusRi0ALNrcqUihhTSZgxS55oHZj4/edit#)
- Check it matches [the design](https://github.com/ubuntudesign/web-squad/issues/266) closely (still to be design +1'ed)
- Check all viewports

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2472

## Screenshots
screen shot 2017-11-29 at 13 13 28-fullpage

### CODE REVIEW

File: templates/financial-services/index.html [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+    <div class="col-12">
+      <h2>Services for the finance industry</h2>
+    </div>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-6 p-card">
+      <div class="p-heading-icon u-no-margin--bottom">
+        <div class="p-heading-icon__header u-no-margin--bottom">
+          <img class="p-heading-icon__img p-heading-icon__img--small" src="{{ ASSET_SERVER_URL }}d6146a84-icon-talk-to-us.svg" alt="" />
+          <h4 class="p-heading-icon__title">Consultation services</h4>
```
Comment (grahambancroft) [inline templates/financial-services/index.html]: Should be h3 with p-heading--four class

File: templates/financial-services/index.html [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+        <p class="u-no-margin--top">
+          <a href="/cloud/foundation-cloud">The Foundation Cloud Build service for a fixed-price cloud built on your premises&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+    </div>
+    <div class="col-6 p-card">
+      <div class="p-heading-icon u-no-margin--bottom">
+        <div class="p-heading-icon__header u-no-margin--bottom">
+          <img class="p-heading-icon__img p-heading-icon__img--small" src="{{ ASSET_SERVER_URL }}034573f2-icon-intro-cloud.svg" alt="" />
+          <h4 class="p-heading-icon__title">Managed cloud services</h4>
```
Comment (grahambancroft) [inline templates/financial-services/index.html]: Should be h3 with p-heading--four class

File: templates/financial-services/index.html [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-6 p-card">
+      <div class="p-heading-icon u-no-margin--bottom">
+        <div class="p-heading-icon__header u-no-margin--bottom">
+          <img class="p-heading-icon__img p-heading-icon__img--small" src="{{ ASSET_SERVER_URL }}f46d02f4-solution-kubernetes.svg" alt="" />
+          <h4 class="p-heading-icon__title">Managed Kubernetes services</h4>
```
Comment (grahambancroft) [inline templates/financial-services/index.html]: Should be h3 with p-heading--four class

File: templates/financial-services/index.html [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+        <p class="u-no-margin--top">
+          <a href="/kubernetes/managed">With the managed Kubernetes service, Canonical will set-up and operate your Kubernetes cluster for you&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+    </div>
+    <div class="col-6 p-card">
+      <div class="p-heading-icon u-no-margin--bottom">
+        <div class="p-heading-icon__header u-no-margin--bottom">
+          <img class="p-heading-icon__img p-heading-icon__img--small" src="{{ ASSET_SERVER_URL }}27141e8b-picto-support-warmgrey.svg" alt="" />
+          <h4 class="p-heading-icon__title">Support</h4>
```
Comment (grahambancroft) [inline templates/financial-services/index.html]: Should be h3 with p-heading--four class

File: templates/financial-services/index.html [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+    <div class="col-12">
+      <h2>Products for the finance industry</h2>
+    </div>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-6 p-card">
+      <div class="p-heading-icon u-no-margin--bottom">
+        <div class="p-heading-icon__header u-no-margin--bottom">
+          <img class="p-heading-icon__img p-heading-icon__img--small" src="{{ ASSET_SERVER_URL }}623e36f9-OpenStack_Logo_Mark+%281%29.svg" alt="" />
+          <h4 class="p-heading-icon__title">OpenStack</h4>
```
Comment (grahambancroft) [inline templates/financial-services/index.html]: Should be h3 with p-heading--four class

File: templates/financial-services/index.html [hunk 6]
```diff
@@ -0,0 +1,239 @@
...
+        <p class="u-no-margin--top">
+          <a href="/openstack">As the number one platform for OpenStack, Canonical supports local clouds and public clouds, at large volume with dynamic scale&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+    </div>
+    <div class="col-6 p-card">
+      <div class="p-heading-icon u-no-margin--bottom">
+        <div class="p-heading-icon__header u-no-margin--bottom">
+            <img class="p-heading-icon__img p-heading-icon__img--small" src="{{ ASSET_SERVER_URL }}f46d02f4-solution-kubernetes.svg" alt="" />
+            <h4 class="p-heading-icon__title">Kubernetes</h4>
```
Comment (grahambancroft) [inline templates/financial-services/index.html]: Should be h3 with p-heading--four class

File: templates/financial-services/index.html [hunk 7]
```diff
@@ -0,0 +1,239 @@
...
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-6 p-card">
+      <div class="p-heading-icon u-no-margin--bottom">
+        <div class="p-heading-icon__header u-no-margin--bottom">
+          <img class="p-heading-icon__img p-heading-icon__img--small" src="{{ ASSET_SERVER_URL }}daeaa851-server.svg" alt="" />
+          <h4 class="p-heading-icon__title">Server provisioning</h4>
```
Comment (grahambancroft) [inline templates/financial-services/index.html]: Should be h3 with p-heading--four class

File: templates/financial-services/index.html [hunk 8]
```diff
@@ -0,0 +1,239 @@
...
+        <p class="u-no-margin--top">
+          <a href="/server">MAAS for automated and optimised provisioning for your production hardware&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+    </div>
+    <div class="col-6 p-card">
+      <div class="p-heading-icon u-no-margin--bottom">
+        <div class="p-heading-icon__header u-no-margin--bottom">
+          <img class="p-heading-icon__img p-heading-icon__img--small" src="{{ ASSET_SERVER_URL }}60d9b81e-picto-canonical.svg" alt="" />
+          <h4 class="p-heading-icon__title">Live automatic security fixes</h4>
```
Comment (grahambancroft) [inline templates/financial-services/index.html]: Should be h3 with p-heading--four class

**Comment by grahambancroft** [comment]
The headings look too close to the text on small, imo 

screen shot 2017-11-29 at 13 50 10

**Comment by anthonydillon** [comment]
@grahambancroft thanks, fixed all the points.

**Comment by grahambancroft** [comment]
Maybe too much space on large now?

screen shot 2017-11-29 at 16 04 19

**Comment by grahambancroft** [comment]
@anthonydillon Fair enough, design review will pick it up if it's an issue I guess

**Comment by kwm14** [comment]
## Visual review

### Services for the finance industry**
- [ ] Update paragraph tag to margin-top: `1rem;`
- [ ] Replace Support icon - https://assets.ubuntu.com/v1/e60711e6-pictogram-support-orange.svg
- [ ] Replace Managed cloud services icon - https://assets.ubuntu.com/v1/b5fde8bb-easy-way-in.svg

### Products for the finance industry
- [ ] Update paragraph tag to margin-top: `1rem;`
- [ ] Replace Server provisioning: https://assets.ubuntu.com/v1/0de4fcd5-logo-maas-icon.svg

### Call to action
- [ ] Update margin-top: `2rem;` on button so it has more space between content above. On medium and small screens it becomes very tight almost on the baseline.

**Comment by anthonydillon** [comment]
@kwm14 Cheers, changes made.

**Comment by kwm14** [comment]
Thanks @anthonydillon looking good 👍

**Comment by frankiegranato** [comment]
My comments are as follows: 

Some small text updates in the copydoc. (all marked @ dev)

Lets add /financial-services under /Cloud in the main nav (this will do for now, but we urgently need to address the IA because all these menus are becoming bloated and confusing)

Thibaut wants us to move the [OFFER] section to lower down the page. See copydoc. I’m a bit worried about doing this because I think it will feel weird, so can we keep one version as it is now? If that is annoying to do, then lets talk :-)  

The headings for the ‘Products for the finance industry’ and ‘Services for the finance industry’ look a bit shouty. Can we reduce the size a bit?

Next to the ‘Managed cloud services’ can we have the cloud with the OpenStack logo in it (not the ubuntu logo please). See example on cloud/managed-cloud.

R3 logo looks a bit big compared to the other logos in the row. Can we reduce it in size? 

@spencerbygraves I don’t like the asset we are using for the Hybrid cloud webinar. It looks a bit odd. Can we find something else? My suggestion: our usual cloud graphic with an ubuntu logo in it (like on currently on /openstack).

**Comment by frankiegranato** [comment]
Just spoke to Thibaut about this page. He wants us to look into using a photograph in the header. Some kind of city scape perhaps.  Not sure how to approach this without a designer. Maybe we just push back this time? What do you think @kasiag & @anthonydillon ? 
Fyi @spencerbygraves

**Comment by spencerbygraves** [comment]
> The headings for the ‘Products for the finance industry’ and ‘Services for the finance industry’ look a bit shouty. Can we reduce the size a bit?

It looks like we use h2 in the same way in the other sections, so I think it's ok.

> Next to the ‘Managed cloud services’ can we have the cloud with the OpenStack logo in it (not the ubuntu logo please). See example on cloud/managed-cloud.

Can we have a single cloud with the OpenStack logo please, I think the managed cloud version wouldn't be very clear at a small size.

>R3 logo looks a bit big compared to the other logos in the row. Can we reduce it in size?

Yes, can it better match the [design](https://github.com/ubuntudesign/web-squad/issues/266#issuecomment-346775192) please. Lyubomir came up with a template to save logo assets at a consistent size, we can use that if we need new assets to make it work.

>I don’t like the asset we are using for the Hybrid cloud webinar. It looks a bit odd. Can we find something else? My suggestion: our usual cloud graphic with an ubuntu logo in it (like on currently on /openstack).

Sure, try that.

**Comment by grahambancroft** [comment]
>Some small text updates in the copydoc. (all marked @ dev)

Done

>Lets add /financial-services under /Cloud in the main nav (this will do for now, but we urgently need to address the IA because all these menus are becoming bloated and confusing)

Done

>Thibaut wants us to move the [OFFER] section to lower down the page. See copydoc. I’m a bit worried about doing this because I think it will feel weird, so can we keep one version as it is now? If that is annoying to do, then lets talk :-)

Done

>I don’t like the asset we are using for the Hybrid cloud webinar. It looks a bit odd. Can we find something else? My suggestion: our usual cloud graphic with an ubuntu logo in it (like on currently on /openstack).

That version won't work on a white background, is there an alternative?

**Comment by grahambancroft** [comment]
Still to do:

- [ ] Next to the ‘Managed cloud services’ can we have the cloud with the OpenStack logo in it (not the ubuntu logo please). See example on cloud/managed-cloud.

- [ ] R3 logo looks a bit big compared to the other logos in the row. Can we reduce it in size?

**Comment by spencerbygraves** [comment]
@grahambancroft this should work for the hybrid image - https://assets.ubuntu.com/v1/7d6041a7-image-cloud.svg

**Comment by grahambancroft** [comment]
Perfect, thank you @spencerbygraves

**Comment by grahambancroft** [comment]
@anthonydillon: approved by @spencerbygraves

**Comment by grahambancroft** [comment]
@anthonydillon only the [photograph](https://github.com/canonical-websites/www.ubuntu.com/pull/2477#issuecomment-348198812) left now

**Comment by anthonydillon** [comment]
@frankiegranato has agreed to live the hero image until we have one ready.

**Comment by frankiegranato** [comment]
I agreed to go live with out the photo in the header but not without addressing my other comments - eg; The offer section didn't move down as requested. This is why we shouldn't remove UX Review labels :-)

**Pull request merged**